tests/istio.io/README.md

```diff
@@ -6,12 +6,66 @@ This folder contains tests for the different examples on Istio.io.
 
 ## Writing Tests
 
+Tests in the Istio.io framework support the following test types:
+
+* Executing scripts using the `RunScript(script string, output outputType, verifier VerificationFunction)` method
+This method accepts an argument which is the name of a script to run, a desired outputType (`TextOutput`,`YamlOutput`, and `JsonOutput`), and an optional verification function. If verifier is set to nil, then any errors from the script will be interpreted as a failure of that step.
+* Applying/Deleting files using `Apply(namespace string, path string)`/`Delete(namespace string, path string)`
+These methods apply and delete the YAML file at path from the specified namespace.
+* Executing custom Go tests by calling `Exec(testFunction testFunc)` where the argument is a function of the form func(t *testing.T) error.
+* Waiting for pods to deploy using the `WaitForPods(fetchFunc KubePodFetchFunc)` function where KubePodFetchFunc follows the form func(env *kube.Environment) kubePkg.PodFetchFunc.
+
+This method allows a test to wait for a set of pods (specified by the fetch function) to start, or for 30 seconds, whichever comes first.
+
+To write a test, add the following imports to your file:
```

Comment:
Please use numbered steps for procedures.

Reply:
Fixed. Thanks.